src/main/java/org/jabref/logic/formatter/Formatters.java

```diff
@@ -63,25 +61,50 @@
             new EscapeUnderscoresFormatter()
     );
 
-    public static final List<Formatter> ALL = new ArrayList<>();
-
     private static final String REGEX = "regex";
 
     private static final int LENGTH_OF_REGEX_PREFIX = REGEX.length();
 
     private Formatters() {
     }
 
+    public static final List<Formatter> getConverters() {
+        List<Formatter> converters = new ArrayList<>();
```

Comment:
same as above

Reply:
In this case it is a defensive copy. The reference to the internal list should not escape the method, because the callers could modify it (which happens in the case of the case changers).